### PR TITLE
finishes exercise 1 curry

### PR DESCRIPTION
**Pull request by teqnickels**

### CODE REVIEW

File: mag-part-1/curry/curry_exercises.js [hunk 1]
```diff
@@ -40,7 +43,7 @@ var max = function(xs) {
   }, -Infinity, xs);
 };
 
-  
+
 // Bonus 1:
```
Comment (pllearns) [inline mag-part-1/curry/curry_exercises.js]: Do you still want to do the `Bonus 1`?

File: mag-part-1/curry/curry_exercises.js [hunk 2]
```diff
@@ -40,7 +43,7 @@ var max = function(xs) {
   }, -Infinity, xs);
```
Comment (pllearns) [inline mag-part-1/curry/curry_exercises.js]: is this supposed to account for a maximum value? (exposing ignorance).

File: mag-part-1/curry/curry_exercises.js [hunk 3]
```diff
@@ -6,9 +6,12 @@ var _ = require('ramda');
 //==============
 // Refactor to remove all arguments by partially applying the function
 
-var words = function(str) {
-  return split(' ', str);
-};
+//ORIGINAL CODE:
```
Comment (pllearns) [inline mag-part-1/curry/curry_exercises.js]: Are you keeping this commented code for future explanation and final project review?

Comment (teqnickels) [inline mag-part-1/curry/curry_exercises.js]: Yes, we are.

File: mag-part-1/curry/curry_exercises.js (hunk 2 above)
Comment (teqnickels) [inline mag-part-1/curry/curry_exercises.js]: - infinity is the lowest minimum value... which there is no limit, hence infinity. xs is the array being passed in. The first time I saw infinity being used was like yesterday. Punit used it in the cog app of Dijkstra's node algorithm (finding the shortest path between two nodes....)

File: mag-part-1/curry/curry_exercises.js (hunk 1 above)
Comment (teqnickels) [inline mag-part-1/curry/curry_exercises.js]: Yes, we completed the bonus 1 and just committed it and sent a prrr request.

**Pull request merged**